wiki/Mapping_Techniques/Formations/en.md

```diff
@@ -144,23 +144,23 @@ The order of the combos forces the player to go along the square rather than thr
 
 Nominators: Beuchi-chan and [mm201](https://osu.ppy.sh/users/30655)
 
-Square formation v2 are pretty much the same as [square formation](#square-formation), but the order of the circles forces you to go diagonally through the square.
+**Square formation v2** are pretty much the same as [square formation](#square-formation), but the order of the circles forces you to go diagonally through the square.
 
 ### Triangle Beats
 
-_Also known as: **Triangle Formation**_
+*Also known as: **Triangle Formation***
 
 Nominator: [kingcobra52](https://osu.ppy.sh/users/9934)
 
-Triangle beats is a very common mapping technique where 3 hit circles are arranged in a triangle.
+**Triangle beats** is a very common mapping technique where 3 hit circles are arranged in a triangle.
```

Comment:
For this one, I'm not sure if we should but "are" insted of "is", but I'm commenting to be sure

Reply:
can be left as is, because "triangle beats" is treated a single term when referred to as one "mapping technique"